.env.example

```
@@ -23,3 +23,8 @@ S3_BUCKET=your-s3-bucket-name
 S3_REGION=us-east-1
 S3_ACCESS_KEY_ID=your-aws-access-key-id
 S3_SECRET_ACCESS_KEY=your-aws-secret-access-key
+
+# Email Configuration with Resend
+RESEND_API_KEY=your-resend-api-key
+FROM_EMAIL=noreply@yourdomain.com
+FROM_NAME=Portfolio CMS
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FROM_NAME key should go before the RESEND_API_KEY key

(UnorderedKey)